packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -120,7 +106,21 @@ contract L2Genesis is Deployer {
     ///         Sets the precompiles, proxies, and the implementation accounts to be `vm.dumpState`
     ///         to generate a L2 genesis alloc.
     function runWithStateDump() public {
-        runWithOptions(OutputMode.DEFAULT_LATEST, artifactDependencies());
+        Fork fork = Config.fork();
+        if (fork == Fork.NONE) {
+            // Will revert if no deploy config can be found either.
+            fork = latestDeployConfigGenesisFork();
+            console.log("L2Genesis: using deploy config fork: %d", uint256(fork));
+        } else {
+            console.log("L2Genesis: using env var fork: %d", uint256(fork));
```

Comment:
It would be way more useful if there was a function that converts the `uint256` to a `string` for these logging statements

Reply:
It would, I just followed the pattern I saw elsewhere where we, instead of implementing string conversions, we just log the enum number representation. But I can add that string converter.

Reply:
Its true that overall we could be better about this 😅 at least in this case, it would be good for chain deployers having insight into the config that is being used

Reply:
We would be able to get rid of `latestDeployConfigGenesisFork` if we have `Config.fork` be the source of truth for both config file + env var